tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -77,15 +77,17 @@ public function testGetPayment() {
       'contribution_id' => $contribution['id'],
     );
 
-    $payment = $this->callAPIAndDocument('payment', 'get', $params, __FUNCTION__, __FILE__);
+    $payment = $this->callAPISuccess('payment', 'get', $params);
 
     $this->assertEquals(1, $payment['count']);
     $expectedResult = array(
-      'total_amount' => 100,
-      'trxn_id' => 23456,
-      'trxn_date' => '2010-01-20 00:00:00',
-      'contribution_id' => $contribution['id'],
-      'is_payment' => 1,
+      $contribution['id'] => array(
+        'total_amount' => 100,
+        'trxn_id' => 23456,
+        'trxn_date' => '2010-01-20 00:00:00',
+        'contribution_id' => $contribution['id'],
+        'is_payment' => 1,
+      ),
     );
     $this->checkPaymentResult($payment, $expectedResult);
```

Comment:
Something is wrong here - the result format should be 

$payment['values'] => array(5 => array(......)

(where 5 is the contribution id)

Why is 'values' missing - that normally gets added by the create_api_success function